src/components/user/CreateTargetForm.js

```diff
@@ -0,0 +1,101 @@
+import React, { PureComponent } from 'react';
+import { func, string, bool, array } from 'prop-types';
+import { Field, reduxForm } from 'redux-form/immutable';
+import {
+  injectIntl,
+  intlShape,
+  defineMessages,
+  FormattedMessage
+} from 'react-intl';
+
+import Loading from 'components/common/Loading';
+import Input from 'components/common/Input';
+import CustomSelect from 'components/common/CustomSelect';
+import { validations, createTarget } from 'utils/constraints';
+// Imagenes
+import createTargetIcon from './../../assets/targetGroup/group.png';
+
+
+const messages = defineMessages({
+  specifyArea: { id: 'target.form.specifyArea' },
+  targetTitle: { id: 'target.form.targetTitle' },
+  selectTopic: { id: 'target.form.selectTopic' }
+});
+
+export class CreateTargetForm extends PureComponent {
+  static propTypes = {
+    topics: array,
+    handleSubmit: func.isRequired,
+    intl: intlShape.isRequired,
+    submitting: bool.isRequired,
+    error: string,
+  }
+
+  mapTopicsToArray() {
+    const { topics } = this.props;
+    const result = [];
+    topics.forEach((item) => {
```

Comment:
This `forEach` where you push something to a new array on each item can be better written using https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Array/map which already returns a new array.

Reply:
It´s done!